packages/react-server/core/logging/server.js

```diff
@@ -1,163 +1,163 @@
-var winston = require('winston')
-,   common  = require('./common')
-,   _ = {
-        mapValues     : require("lodash/object/mapValues"),
-        pick          : require("lodash/object/pick"),
-        isPlainObject : require("lodash/lang/isPlainObject"),
-        isEmpty       : require("lodash/lang/isEmpty"),
-        trimLeft      : require("lodash/string/trimLeft"),
-        trunc         : require("lodash/string/trunc"),
-    };
-
-var makeLogger = function(group, opts){
-	var config = common.config[group];
-
-	var fileTransport = new (winston.transports.File)({
-		name      : 'file',
-		level     : config.baseLevel,
-		stream    : process.stdout,
-		json      : false,
-		timestamp : global.TIMESTAMP_REACT_SERVER_LOG_OUTPUT,
+var	winston	=	require('winston')
+,			common		=	require('./common')
+,			_	=	{
+	mapValues					:	require("lodash/object/mapValues"),
+	pick										:	require("lodash/object/pick"),
+	isPlainObject	:	require("lodash/lang/isPlainObject"),
+	isEmpty							:	require("lodash/lang/isEmpty"),
+	trimLeft						:	require("lodash/string/trimLeft"),
+	trunc									:	require("lodash/string/trunc"),
```

Comment:
Whoa, and now that I see this on the main PR page the tabling is all off... is it using _tabs_ for tabling?  That's generally not a good idea.  People set their tab stops at various positions.  Tabs work for _initial_ whitespace, but they don't work well for whitespace within a line.